CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.1.0] - 2017-07
```

Comment:
Normally the versioning and release date is left to the maintainer to bump as there is no guarantee on merge order in this case is something I can release tonight I am gonna allow it this time but next time just put sub headers for the type of change under unreleased.